scripts/create_pmid_pkl.py

```diff
@@ -0,0 +1,36 @@
+import click
+import pickle
+
+from sympy.logic.boolalg import Boolean
+
+from scripts.utils import find_pmids
+
+
+@click.command(
+    help="""
+This takes a directory as an input, searches for strings of the form PMID_{1-8 digits} among the file names
+and writes those PMIDS to a .pkl file. 
+
+PMID_DIRECTORY:            The directory where you want to search for PMIDs among the file names.
+PKL_FILE_PATH:             The file path of the .pkl file which will contain those PMIDs as a set.
+RECURSIVE DIR SEARCH:      If True then the search through PMID_directory looks through subdirectories.
+
+Example:
+assets/cases assets/pmids.pkl
+"""
+)
+@click.argument("pmid_directory", type=click.Path(exists=True, dir_okay=True))
+@click.argument("pkl_file_path", type=click.Path(exists=False))
```

Comment:
There should be a file_ok parameter or something in the path object and a dir_ok.

Think file_ok=True, while dir_ok=False